src/transformers/models/rag/modeling_rag.py

```diff
@@ -1633,6 +1633,8 @@ def extend_enc_output(tensor, num_beams=None):
             exponential_decay_length_penalty=exponential_decay_length_penalty,
             logits_processor=logits_processor,
             renormalize_logits=renormalize_logits,
+            supress_tokens=None,  # Not used for RAG, were added after the whisper pull request
+            begin_supress_tokens=None,  # Not used for RAG, were added after the whisper pull request
```

Comment:
```suggestion
            suppress_tokens=None,  # Not used for RAG, were added after the whisper pull request
            begin_suppress_tokens=None,  # Not used for RAG, were added after the whisper pull request
```
Same here

Reply:
Yes, those `None` should just be the default in the base method. This file should not be modified.

Reply:
Yes agree! let's default them to `None` in the base method so we don't need any changes here